package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "schulcloud-nuxt",
-	"version": "26.5.1",
+	"version": "26.5.2",
```

```suggestion
	"version": "26.5.1",
```
probably we can leave that for now at 26.5.1